pom.xml

```diff
@@ -73,7 +73,7 @@
   <org.apache.commons.validator.version>1.7</org.apache.commons.validator.version>
   <org.apache.commons.io.version>2.8.0</org.apache.commons.io.version>
   <org.apache.httpcomponents.version>4.5.13</org.apache.httpcomponents.version>
-  <io.fabric8.client.version>5.4.1</io.fabric8.client.version>
+  <io.fabric8.client.version>5.12.3</io.fabric8.client.version>
```

Comment:
Are we able to use 5.12.2 instead to sync with Quarkus 2.7? https://github.com/quarkusio/quarkus/blob/8ecbe2c5282b26227139798e610cb98a176db974/bom/application/pom.xml#L171

Reply:
Done in last push, seems to be working fine.